spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkLocalScan.java

```diff
@@ -50,8 +55,13 @@ public StructType readSchema() {
 
   @Override
   public String description() {
-    String fields =
-        Arrays.stream(readSchema.fields()).map(StructField::name).collect(Collectors.joining(", "));
-    return String.format("%s [%s]", table, fields);
+    return String.format("%s [filters=%s]", table, Spark3Util.describe(filterExpressions));
+  }
+
+  @Override
+  public String toString() {
+    return String.format(
+        "IcebergScan(table=%s, type=%s, filters=%s)",
```

Comment:
What about `IcebergScan` -> `IcebergLocalScan` to indicate it is a local scan?

Reply:
Fixed. Thanks